### PR TITLE
refactor: Update values

### PR DESCRIPTION
**Pull request by joaocosta-codacy**

### CODE REVIEW

File: codacy/values.yaml [hunk 1]
```diff
@@ -252,8 +252,9 @@ global:
   metrics:
     serviceMonitor:
       enabled: false
-    grafana_dashboards:
+    grafana:
```
Comment (paulopontesm) [inline codacy/values.yaml:255]: didn't this change to global?

**Comment by paulopontesm** [comment]
we need to update this path:
	ytool -f "./codacy/values.yaml" -s worker-manager.config.codacy.worker.imageVersion "${ENGINE_VERSION}" -e

File: codacy/values.yaml (hunk 1 above)
Comment (joaocosta-codacy) [reply codacy/values.yaml:255]: but this is the global block. I am confused. :)

Comment (paulopontesm) [reply codacy/values.yaml:255]: sorry .. i'm drunk🤦‍♂️

**Pull request merged**